### ISSUE TITLE
Disc change option for the browser

### ISSUE THREAD

**Issue by sDetective** [kootstra-rene/enge-js#31]
can you add a disc change option for the browser it helps with the other games like mgs.

**Comment by kootstra-rene** [comment]
indeed a good suggestion will need to investigate this request, do you have any save states just before a disc swap? or know where to find them?

**Comment by sDetective** [comment]
Yeah, I do sorry for the very late response I've been doing a ton of stuff recently but anyway I may have a save state its been a while ill have to play through it again.

**Comment by sDetective** [comment]
I don't have a save all I have is where the trigger would be, here's the youtube link for it where it would be so you would change discs https://www.youtube.com/watch?v=9j_Pd71-5J0. Are there any hotkeys for saving I didn't know like saving in eGNE?

**Comment by kootstra-rene** [comment]
I'll see if I can find some tests otherwise I'll just have to keep playing FFVII until a discswap.

**Comment by Platinumbit** [comment]
No update?

**Comment by kootstra-rene** [comment]
Sadly no, the emulator has really low importance now.

**Comment by Platinumbit** [comment]
Will you ever update it with a disc change?

**Comment by kootstra-rene** [comment]
Probably, because it is an important addition. Though with FFVI someone was able to finish be just changing the disc and restart the emulator to load the latest save point.

**Comment by Platinumbit** [comment]
Interesting, I'm trying to play Metal Gear Solid on it, and I wonder if it will work. Only one way to find out
Use 6-7 hours of my life to do it lol

**Comment by kootstra-rene** [comment]
Or find a save point online just before the disc change? but loading that is somewhat tedious... never mind

**Comment by Platinumbit** [comment]
True... But MGS is fun as heck to play lol
I'll try to find a save online
If I can't
Metal Gear

**Comment by Platinumbit** [comment]
I tried the thing. Not happening. Guess Ill find a save file before Metal Gear REX.

**Comment by Platinumbit** [comment]
Uhh.. how do I insert a saved data file
